overlay.d/07c9s/bootstrap/pre-pivot.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Binaries shipped as part of the ostree image need to be in `/usr` (not `/usr/local`).  So *if* we were to ship this, it could be e.g. `/usr/libexec/ocp-node-bootstrap.sh` or so.

But I'm questioning whether we really need this tiny shell script in the OS versus injected via Ignition to the bootstrap node, which is where half of this logic is anyways.

Reply:
FCOS' layered image ships this script in `/bootstrap/pre-pivot.sh` today, and I wanted to re-use as much of that as possible.
You are right though, that we could just add this in the installer directly as ignition. I'll fold the bootstrap pivot into https://github.com/openshift/installer/pull/6443